.github/workflows/pr.yml

```diff
@@ -293,7 +354,7 @@ jobs:
           body="### Add dataset(s) - $first_collection_id [Automated PR by ${{ github.actor }}]\n\n$collection_bullet_points"
```

Comment:
I left the github.actor here. I think it's nice to see the user that opened the original veda-data PR with the dataset-config changes. Thoughts welcome, though! I'm definitely okay with removing it.